src/hooks/listHooks.js

```diff
@@ -62,6 +62,8 @@ export const useRenderItem = (query, navigation, options = {}) => {
     }
     default: {
       renderItem = ({ item, index, section }) => {
+        if (query === QUERY_TYPES.CONSUL.PROPOSALS && !item.published) return;
```

Comment:
good idea. but i think the list data should be cleaned up instead of rendering nothing for some entries here. there should be only list data in the array passed to the list component that we want to render on the screen. i guess the `listItems` in the `ConsulIndexScreen` need to be modified base on `item.published` and all entries could be rejected, that are draft. the list item rendering would not need to be touched. the logics are already very heavy here deciding which list item component to use. so we should not add another complexity with checking for specific data to do something event more different. what do you think?

Reply:
I guess I can add a filter for `Proposals`. If the `item.pulished` value is `true`, it will be displayed in the listing process on the screen.

Reply:
nice, very good solution!